README.md

```diff
@@ -54,4 +58,33 @@ To download MATLAB go [here](https://www.mathworks.com/downloads/).
 ### Python
 We will be using Python 3.6 with the Anaconda distribution. This is a large package that includes many of the base functions and IDE's that are very common to Python.
 
-To download the Anaconda distribution, go [here](https://www.anaconda.com/products/individual) and choose your installer near the bottom. Anaconda comes with a few possible IDE's, we prefer VScode but you can use whichever to edit the scripts and run them.
\ No newline at end of file
+To download the Anaconda distribution, go [here](https://www.anaconda.com/products/individual) and choose your installer near the bottom. Anaconda comes with a few possible IDE's, we prefer VScode but you can use whichever to edit the scripts and run them.
+
+## 4. Creating the Git Repository for your group.
+This section should cover how the group leader will download the repository, create their own version of it, and then invite their group members (collaborators) to the project. Only one person per group needs to do this.
```

Comment:
add something like "if you haven't been asked to do this, don't." so it's very clear that those who "need to do this" know that they need to do it.

Reply:
or use some ❗ or 🚨 emojis. Need it to be very clear that 99% of the attendees should NOT do this. could consider hiding it in the wiki.

Reply:
Emoji's.... i'ma go with no lol.